frontend/guardian.rb

```diff
@@ -98,30 +99,14 @@ def rejected_beep(gpio_command)
     tag = $2
   end
 
-  tag_log = {
+  tag_log = TagLog.create(
     card_type: tag =~ /^c/ ? 'clipper' : 'rfid',
     card_number: tag,
     held_tag: flag == 'h' ? true : false
-  }
-
-  authorization = DoorAuthorization.first(card_number: tag)
-
-  if authorization
-    tag_log[:name] = authorization.name
-
-    unless authorization.expired? || ! authorization.active?
-      tag_log[:door_opened] = true
-    end
-  end
-
-  TagLog.create(tag_log)
+  )
 
-  if tag_log[:door_opened]
-    # open the door
-    open_door(gpio_command)
-    accepted_beep(gpio_command)
-    close_door(gpio_command)
-  else
-    rejected_beep(gpio_command)
+  if tag_log.door_opened
+    open_door(gpio_command, tag_log.door)
```

Comment:
I would continue the practice of having a .1s delay between the time you bring the pin high and the time you bring the pin low again.  The accept_beep had a .1s delay inside it that meant you didn't flip the state of the line quickly which helps eliminate any potential timing issues on the opening circuit.  (Everything is likely work without this since the pi isn't that fast and we're shelling out to execute a command on an entirely new thread, but since we have additional latency in the wire and circuit to the rollup, let's keep giving things that ridiculously generous .1s to settle as we did before.)  Should be easy enough to do just by replacing the accepted_beep call to a sleep .1 call.